lib/utils.js

```diff
@@ -18,16 +18,17 @@ const DEFAULT_SIM_SHUTDOWN_TIMEOUT = 10000;
 // 2       Invalid options were specified on the command line.
 // 3       An internal error occurred.
 
-async function forceKillAll (appName) {
+async function killApp (appName, isForceKill = false) {
+  let args = ['-x', appName];
+  if (isForceKill) {
+    args.splice(0, 0, '-9');
+  }
   try {
-    await exec('pkill', ['-9', '-x', appName]);
+    await exec('pkill', args);
+    return 0;
   } catch (e) {
-    if (e.code === 1) {
-      log.debug(`${appName} is not running. Continuing...`);
-      return;
-    }
-    log.error(`pkill error ${e.code} while trying to forcefully kill ${appName}`);
-    throw e;
+    if (e.code) return e.code;
+    return e;
```

Comment:
I never see us doing anything with a returned `Error` from this function; do we not want to `throw` if we can't find a code?